addon/components/bootstrap/-form-controls.js

```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
+import layout from '../../templates/components/bootstrap/-form-controls';
+
+export default Ember.Component.extend({
+  layout
```

Comment:
is this breaking? Why is layout there?

Reply:
Same as above

Reply:
If I understand correctly @harokevin the `import` is only used if it is called. So this is importing the layout, then calling/applying it, even though it doesnt need to do anything except BE a layout. Someone please correct me if I'm wrong.